From be8b4cdf2939d5cd2bf3f2a38534f0d760f63413 Mon Sep 17 00:00:00 2001
From: jmckenna <jason.mckenna@windriver.com>
Date: Fri, 6 Oct 2017 10:38:33 -0400
Subject: [PATCH] CGTS-6766: Openstack client logging a warning message when
 'admin'     user password is changed using 'openstack user set'CLI  command


diff --git a/openstackclient/identity/v3/user.py b/openstackclient/identity/v3/user.py
index 5f4fb54..201d217 100644
--- a/openstackclient/identity/v3/user.py
+++ b/openstackclient/identity/v3/user.py
@@ -392,6 +392,10 @@ class SetUser(command.Command):
             kwargs['enabled'] = False
 
         identity_client.users.update(user.id, **kwargs)
+        if user.name == 'admin' :
+            print("Warning: '%s' password changed. Please wait 5 minutes "
+                  "before Locking/Unlocking the controllers for the password "
+                  "change to come into effect\n" %(user.name))
 
 
 class SetPasswordUser(command.Command):
